packages/esm-patient-search-app/src/config-schema.ts

```diff
@@ -7,6 +7,11 @@ export const configSchema = {
       _description: 'Where clicking a patient result takes the user. Accepts template parameter ${patientUuid}',
       _validators: [validators.isUrlWithTemplateParameters(['patientUuid'])],
     },
+    showRecentlySearchedPatients: {
+      _type: Type.Boolean,
+      _default: false,
```

Comment:
I think it's most likely that we should leave the default as `true`, since that's effectively the functionality we've been shipping since this feature was added.

Reply:
I'm inclined to keep this as `false`, at least until I've taken a stab at improving the performance of the Recent Patient search functionality (which I intend to do following this PR).